src/components/Navbar.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import '../styles/Navbar.css';
+
+function Navbar({ home, calculator, quote }) {
+  return (
+    <nav>
+      <h2 className="logo">Math Magicians</h2>
+      <ul className="nav-item">
+        <li className="nav-item-list"><Link to={home}>Home</Link></li>
+        <li className="nav-item-list"><Link to={calculator}>Calculator</Link></li>
+        <li className="nav-item-list"><Link to={quote}>Quote</Link></li>
```

Comment:
- Optional
I would recommend using `NavLink` instead of `Link`. You can find the difference between the two[ here](https://www.educative.io/answers/what-are-the-differences-between-navlink-link-and-a-in-react).